packages/shared/lib/core/wallet/utils/generateActivity/helper/getAsyncDataFromOutput.ts

```diff
@@ -14,14 +15,15 @@ export async function getAsyncDataFromOutput(
 ): Promise<AsyncData> {
     const isAsync = isOutputAsync(output)
     if (isAsync) {
+        const commonOutput = output as CommonOutput
         const isClaiming = false
         const claimingTransactionId = claimingData?.claimingTransactionId
         const claimedDate = claimingData?.claimedDate
         const isRejected = isActivityHiddenForAccountIndex(account.index, outputId)
 
-        const expirationDate = getExpirationDateFromOutput(output)
-        const timelockDate = getTimelockDateFromOutput(output)
-        const { storageDeposit } = await getStorageDepositFromOutput(account, output)
+        const expirationDate = getExpirationDateFromOutput(commonOutput)
+        const timelockDate = getTimelockDateFromOutput(commonOutput)
```

Comment:
`expirationDate` and  `timelockDate` are now `Date | undefined` so we have type error on line 38 and 39.